maths/euclidean_distance.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+
+def euclidean_distance(v1, v2) -> double:
+    """
+    Calculate the distance between the two = endpoints of two vectors, v1 and v2.
+    >>> euclidean_distance(np.array([0, 0]), np.array([2, 2]))
+    2.8284271247461903
+    >>> euclidean_distance(np.array([0, 0, 0]), np.array([2, 2, 2]))
+    3.4641016151377544
+    >>> euclidean_distance(np.array([1, 2, 3, 4]), np.array([5, 6, 7, 8]))
+    8.0
+    """
+
+    sumSquared = 0 
+    for i in range(len(v1)):
+        sumSquared += (v2[i] - v1[i]) ** 2
+    return sumSquared ** (1 / 2)
```

Comment:
You don't need to loop. You can use a one-liner. 
`numpy.sqrt(numpy.sum((x-y)**2))`

You can also use an equivalent linear algebra trick
```
diff = point_a - point_b
numpy.dot(diff.T, diff) ** (1/2)
```

I will leave it to you verify the correctness of the above. xD Don't take my math blindly.